backend/data_preparation/crawler/twitter_filter_api_crawler.py

```diff
@@ -60,18 +61,21 @@ def crawl(self, keywords: List, batch_number: int = 100) -> List[int]:
                     self.reset_wait_time()
 
                     has_keywords = set(self.keywords) & self._tokenize_tweet_text(tweet)
```

Comment:
this line did not use keywords in lower case. DO change the self.keywords to be contain lower case only